base/rational.jl

```diff
@@ -408,9 +408,9 @@ function *(y::Integer, x::Rational)
     yn, xd = divgcd(promote(y, x.den)...)
     unsafe_rational(checked_mul(yn, x.num), xd)
 end
-/(x::Rational, y::Union{Rational, Integer, Complex{<:Union{Integer,Rational}}}) = x//y
-/(x::Union{Integer, Complex{<:Union{Integer,Rational}}}, y::Rational) = x//y
-inv(x::Rational{T}) where {T} = checked_den(x.den, x.num)
+/(x::Rational, y::Union{Rational, Integer}) = x//y
+/(x::Integer, y::Rational) = x//y
+inv(x::Rational) = checked_den(x.den, x.num)
```

Comment:
```suggestion
inv(x::Rational{T}) where {T} = checked_den(x.den, x.num)
```

This change is not related, right? IMO unrelated cosmetic changes should go separate from bugfix PRs, making both reviewing and `git` cherry-picking easier.